gridlab-d/gridlab-d
# (#1220) Implement ability to dump rhs of matrix equations for use in external programs

### What's this Pull Request do?
Updates the matrix output methods in NR to include the RHS information

#### Where should the reviewer start?
Compile the code and run an example, such at powerflow/test_NR_matrix_dump.glm.  Make sure "NR_matrix_output_rhs" is true and hit enter.

#### How should this be tested?
Run the GLM above with teh "NR_matrix_output_rhs" set, and see if the output file reflect thta.

#### Any background context you want to provide?
Similar to legacy #992, but that code is really old (and does something slightly different)

#### What are the relevant issues?
N/A

#### Questions:
